src/NUnitFramework/tests/Constraints/CollectionEqualsTests.cs

```diff
@@ -72,5 +75,32 @@ public void HonorsIgnoreCase( IEnumerable expected, IEnumerable actual )
             new object[] {new List<string> {"a", "b", "c"}, new List<string> {"A", "B", "C"}},
         };
 
+        [Test]
+        public void DoesContainMultipleItemsString()
+        {
+            var collection = new[] { "test1", "test2", "test3" };
+            Assert.That(collection, Does.Contain("test1").And.Contains("test2").And.Contains("test3"));
+        }
+
+        [Test]
+        public void ContainsMultipleItemsString()
+        {
+            var collection = new[] { "test1", "test2", "test3" };
+            Assert.That(collection, Contains.Item("test1").And.Contains("test2").And.Contains("test3"));
+        }
+
+        [Test]
+        public void DoesContainMultipleItemsInt()
+        {
+            var collection = new[] { 1, 2, 3 };
+            Assert.That(collection, Does.Contain(1).And.Contains(2).And.Contains(3));
+        }
+
+        [Test]
+        public void ContainsMultipleItemsInt()
+        {
+            var collection = new[] { 1, 2, 3 };
+            Assert.That(collection, Contains.Item(1).And.Contains(2).And.Contains(3));
+        }
```

Comment:
Tests look good, but seems weird to put them in `CollectionEqualsTests`. Do we have other tests on (collection) Contains already?

Reply:
Agreed, I'd looked for a better place, but didn't find one. I went back and the best I can find is a fixture called `ListContentTests` so I am moving there.